crowdsourcehinter/crowdsourcehinter.py

```diff
@@ -1,19 +1,20 @@
+from __future__ import absolute_import
 import logging
 import operator
 import pkg_resources
 import random
 import json
 import copy
 
-import urllib
-import HTMLParser
+import six.moves.urllib.request, six.moves.urllib.parse, six.moves.urllib.error
```

Comment:
The `request` and `error` imports seem unused, they can probably be dropped.  I'd be inclined to use `from six.moves.urllib.parse import unquote_plus` instead to make the code below marginally cleaner, but it probably doesn't matter that much.